pyqtorch/embed.py

```diff
@@ -0,0 +1,131 @@
+from __future__ import annotations
+
+from logging import getLogger
+from typing import Any, Callable
+
+import torch
+
+logger = getLogger(__name__)
+
+
+def torch_call(
```

Comment:
Would it make sense to rename smt like `call_to_torch` ? I find `torch_call` a bit misleading. Or simply `torchify` ?

Reply:
What worries me here is that it seems easy to misuse this function and break it. Say for instance, what if you pass a wrong list of arguments ? It would be difficult to check correctness for every arbitrary `abstract_fn`.